lib/index.ts

```diff
@@ -1,8 +1,9 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import * as graphlib from '@snyk/graphlib';
 import * as tmp from 'tmp';
 import debugLib = require('debug');
```

Comment:
Debug has types and can be imported like a module

Reply:
This is not relevant to this pull request (the code was already there) if you want you can raise a side pr where you upgrade it

Reply:
It's related to overall quality of code 🤗

Reply:
I understand your point of view here, I just want to keep it simple.

There are different things here that we can improve (basic one lint) but we can do that afterwards ^^